version.txt

```diff
@@ -1 +1 @@
-1.30.0-beta.0
\ No newline at end of file
+1.31.0
```

Comment:
you need to add the `-rc.0` or `-beta.0` etc suffix here, and also add the `k8s.io` dependency changes